README.md

```diff
@@ -2,179 +2,128 @@
 
-- for linux: [3A3) Recommended: Upgrade gtp2ogs.js from old branch to “devel” branch (latest)](https://github.com/wonderingabout/gtp2ogs-tutorial/blob/master/docs/3A3-linux-optional-upgrade-to-devel.md)
-- for windows: [3B3) Recommended: Upgrade gtp2ogs from old branch to devel (latest) branch](https://github.com/wonderingabout/gtp2ogs-tutorial/blob/master/docs/3B3-windows-optional-upgrade-to-devel.md)
+After that, it is mandatory to also yout AI settings. To do that, add a `--`
+ (with one space before and one space after the `--` ), to separate `<gtp2ogsarguments>`
+ from your bot path and `<botarguments>`.
 
-### 4. Most common usage: start gtp2ogs.js using nodejs
+## Beta OGS
 
-For linux (preferably as sudo):
+Before you can play on official [OGS server](https://online-go.com/) server, OGS admins
+ will often ask that you run tests to make sure everything is running fine.
 
-```Shell
-node /usr/lib/node_modules/gtp2ogs/gtp2ogs.js --username <yourbotusername> --apikey <apikey> <gtp2ogsargument1> <gtp2ogsargument2> -- /replace/with/full/path/to/your/ai/runfile.file <botargument1> <botargument2>
-```
+You need to do these tests in [beta OGS server](https://beta.online-go.com/).
 
-For windows (preferably as admin):
+When all testing is OK, you can remove the `--beta` and play in the real OGS.
 
-```Shell
-pushd C:\Program Files\nodejs && node.exe C:\Users\yourwindowsusername\AppData\Roaming\npm\node_modules\gtp2ogs\gtp2ogs.js --username <yourbotusername> --apikey <apikey> <gtp2ogsargument1> <gtp2ogsargument2> -- C:\Users\path\to\your\ai\executable.exe <botargument1> <botargument2>
-```
+## Example of use
 
-note: for all operating systems, in `--`, the spaces after `<gtp2ogsarguments>`
-and before `/path/to/your/bot.executable` are important: they separate gtp2ogs
-arguments from your bot arguments
+In this example AI is PhoenixGo, but it works in a similar way for any AI.
 
-note 2: the number of `<gtp2ogsarguments>` and `<botarguments>` is not limited,
-here only 2 were shown but it possible to use for example 3,4,5 , or as many as
-you want
-  
-note 3: to play on [beta OGS server](https://beta.online-go.com/) instead of the
-[OGS server](https://online-go.com/), add the `--beta` argument
+The number of gtp2ogs options and of bot arguments is free.
 
-## Options
+### For Windows
 
-You can find a list of all possible to use gtp2ogs arguments [here](/docs/OPTIONS-LIST.md).
+General use case is something like this:
 
-Before putting `<gtp2ogsarguments>`, you have to put the options [username](https://github.com/online-go/gtp2ogs/blob/devel/docs/OPTIONS-LIST.md#username)
-and [apikey](https://github.com/online-go/gtp2ogs/blob/devel/docs/OPTIONS-LIST.md#apikey)
-first.
+`
+pushd C:\Program Files\nodejs\node.exe C:\Users\yourusername\gtp2ogs-node\node_modules\gtp2ogs\gtp2ogs.js --username <yourbotusername> --apikey <apikey> <gtp2ogsargument1> <gtp2ogsargument2> -- C:\Users\path\to\your\ai\executable.exe <botargument1> <botargument2>
```

Comment:
i would make these example command as small as possible

Reply:
i removed a few commands (bans (since we dont always use these), maxconnectedgames and maxconnectedgamesperuser (since we provide defaults for these and are unlikely to want to change the defaults), i also added ogspv to highlight that it exists

for the other options i thought it would be a good way to illustrate a real use case of gtp2ogs to show the range of possibilities to a new user